website/docs/docs/dbt-versions/core-upgrade/05-upgrading-to-v1.10.md

```diff
@@ -37,7 +37,7 @@ With [hybrid projects](/docs/deploy/hybrid-projects), <Constant name="core"/> us
 
 - Fostering collaboration between <Constant name="cloud"/> + <Constant name="core"/> users by enabling them to visualize and perform [cross-project references](/docs/collaborate/govern/project-dependencies#how-to-write-cross-project-ref) to models defined in <Constant name="core"/> projects. This feature unifies <Constant name="cloud"/> + <Constant name="core"/> workflows for a more connected dbt experience.
 - Giving <Constant name="cloud"/> and <Constant name="core"/> users insights into their models and assets in [<Constant name="explorer"/>](/docs/collaborate/explore-projects). To view <Constant name="explorer"/>, you must have have a [developer or read-only license](/docs/cloud/manage-access/seats-and-users).
-- (Coming soon) Enabling users working in the [<Constant name="visual_editor"/>](/docs/cloud/visual-editor) to build off of models already created by a central data team in <Constant name="core"/> rather than having to start from scratch.
+- (Coming soon) Enabling users working in the [<Constant name="visual_editor"/>](//docs/cloud/canvas) to build off of models already created by a central data team in <Constant name="core"/> rather than having to start from scratch.
```

Comment:
```suggestion
- (Coming soon) Enabling users working in the [<Constant name="visual_editor"/>](/docs/cloud/canvas) to build off of models already created by a central data team in <Constant name="core"/> rather than having to start from scratch.
```